electron-app/src/components/display/AmountDisplay.vue

```diff
@@ -51,16 +51,14 @@ import { Currency } from '@/model/currency';
 import { bigNumberify } from '@/utils/bignumbers';
 
 const { mapGetters } = createNamespacedHelpers('session');
+const { mapState } = createNamespacedHelpers('session');
```

Comment:
```suggestion
const { mapState, mapGetters } = createNamespacedHelpers('session');
```
No need to do this twice

Reply:
Fixed in 284e11b